lms/djangoapps/grades/tasks.py

```diff
@@ -147,6 +155,54 @@ def recalculate_course_and_subsection_grades_for_user(self, **kwargs):  # pylint
         )
 
 
+@task(
+    bind=True,
+    base=LoggedTask,
+    time_limit=MINIMAL_GRADE_REACHED_NOTIFICATION_TIMEOUT_SECONDS,
+    default_retry_delay=RETRY_DELAY_SECONDS,
+    routing_key=settings.POLICY_CHANGE_GRADES_ROUTING_KEY,
+)
+def notify_student_about_course_is_graded(self, user_id, course_id, **kwargs):
+    """
+    Send an email informing that the user has reached a minimal grading for the course.
+    """
+    user = User.objects.filter(id=user_id).first()
+    key = CourseKey.from_string(course_id)
+    course = CourseOverview.objects.filter(id=key).first()
+    context = {
+        'full_name': user.profile.name,
+        'platform_name': configuration_helpers.get_value("PLATFORM_NAME", settings.PLATFORM_NAME),
+        'course_name': course.display_name,
+    }
+    context['site'] = configuration_helpers.get_value(
+        'SITE_NAME',
+        settings.SITE_NAME
+    )
```

Comment:
@dyudyunov I guess there's no reason to separate the assignment of `site` from general project creation. Like so
```
    'site': configuration_helpers.get_value('SITE_NAME', settings.SITE_NAME),
}
```

Reply:
Agree, thx.